server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
Fair point - OTOH are we ever likely to package hobknob as a separate npm module?